src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -828,6 +866,32 @@ class TDatabaseFeedManager final : public Observer<nlohmann::json&>
         // LCOV_EXCL_STOP
     }
 
+    void fileProcessingStatusReport()
+    {
+        m_fileProcessingStatusThread = std::thread(
+            [this]()
+            {
+                m_fileProcessingStatusServer.Get(
+                    "/status",
+                    [&](const httplib::Request&, httplib::Response& res)
+                    { res.set_content(m_fileProcessInProgress.load() ? "Active" : "Inactive", "text/plain"); });
```

Comment:
Improve (design): We should define constants for the "Active" and "Inactive" response values

Reply:
Changed

Reply:
Improve (design): If `FILE_PROCESSING_ACTIVE_STATUS` changes to other value, this will start to fail 

```suggestion
                    { res.set_content(m_fileProcessInProgress.load() ? FILE_PROCESSING_ACTIVE_STATUS : FILE_PROCESSING_INACTIVE_STATUS, "text/plain"); });
```